plugins/user-settings/api-report.md

```diff
@@ -116,15 +140,6 @@ export class UserSettingsStorage implements StorageApi {
   snapshot<T extends JsonValue>(key: string): StorageValueSnapshot<T>;
 }
 
-// @public
-export const UserSettingsTab: (props: UserSettingsTabProps) => JSX.Element;
-
-// @public (undocumented)
-export type UserSettingsTabProps = PropsWithChildren<{
-  path: string;
-  title: string;
-}>;
```

Comment:
I think we should keep this in place but deprecate it. It's a bit unnecessary to make it breaking like this. Let's just update the `USER_SETTINGS_TAB_KEY` to match the new one that `SettingsLayout.Route` uses.

Reply:
Sorry, misunderstood you the first time. Added `UserSettingsTab` back with a `@deprecation` annotation. Also had to replace its internals to use `SettingsLayout.Route`, added some tests to check for contents of the tab